scripts/install_golang.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -euo pipefail
+
+GO_VERSION="${GO_VERSION:-1.17.4}"
+
+function install_golang() {
+    if ! [ -x "$(command -v go)" ]; then
+        echo "Installing golang..."
+        curl -s https://go.dev/dl/go$GO_VERSION.linux-amd64.tar.gz | tar -C /usr/local -xz
```

Comment:
local is enough? not always worked for me

Reply:
It works